runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -132,11 +136,24 @@ private void setNativeContent(long newNativeContent) {
 
         assert mNativeContent == 0 && mCleanupReference == null && mContentViewCore == null;
 
+        boolean animated;
+
         // Initialize ContentViewRenderView
-        boolean animated = XWalkPreferencesInternal.getValue(
+        if (attrs == null) {
+            animated = XWalkPreferencesInternal.getValue(
                 XWalkPreferencesInternal.ANIMATABLE_XWALK_VIEW);
+        } else {
+            //This means developer wants to assign the CompositingSurfaceType for a
```

Comment:
Style: remember that there should be a space before the first word in each line.